mmhuman3d/models/utils/inverse_kinematics.py

```diff
@@ -66,10 +66,15 @@ def batch_inverse_kinematics_transform(pose_skeleton,
     rel_pose_skeleton[:, 0] = rel_rest_pose[:, 0]
 
     # the predicted final pose
-    final_pose_skeleton = torch.unsqueeze(pose_skeleton.clone(), dim=-1)
-    final_pose_skeleton = final_pose_skeleton - \
-        final_pose_skeleton[:, 0:1] + rel_rest_pose[:, 0:1]
-
+    if train:
+        final_pose_skeleton = torch.unsqueeze(pose_skeleton.clone(), dim=-1)
+        final_pose_skeleton[:, 1:] -= final_pose_skeleton[:,
+                                                          parents[1:]].clone()
+        final_pose_skeleton[:, 0] = rel_rest_pose[:, 0]
+    else:
+        final_pose_skeleton = torch.unsqueeze(pose_skeleton.clone(), dim=-1)
```

Comment:
It seems that L70 and L75 share the same code, thus we may put it before the if-else.